kanopi/launch_checklist
# 26772586 Adds Checklist Items [WIP]

## Description
> As a developer, I need to create a Drupal 8+ module that mimics the Kanopi checklist.

## Affected URL
N/A

## Related Tickets
* [Drupal 8+ Launch Checklist module](https://kanopi.teamwork.com/#/tasks/26772586)

## Steps to Validate
1. Install this module on a Drupal 8+ site
2. Navigate to the Checklist in the admin.
3. Verify checklists appear
4. Verify links work.

## Deploy Notes
N/A
